src/components/sprite-selector/sprite-selector.jsx

```diff
@@ -0,0 +1,57 @@
+const React = require('react');
+
+const SpriteSelectorItem = require('../../containers/sprite-selector-item.jsx');
+
+const styles = require('./sprite-selector.css');
+
+const SpriteSelectorComponent = function (props) {
+    const {
+        onSelectSprite,
+        selectedId,
+        sprites,
+        ...componentProps
+    } = props;
+    return (
+        <div
+            className={styles.spriteSelector}
+            {...componentProps}
+        >
+            {Object.keys(sprites)
+                // Only render ready sprites
+                .filter(id => 'costume' in sprites[id] && 'order' in sprites[id])
```

Comment:
can you explain this line a little more? do you know why some sprites are missing these properties?